halo/app/app.go

```diff
@@ -278,6 +278,10 @@ func (a App) GetSlashingKeeper() slashingkeeper.Keeper {
 	return a.SlashingKeeper
 }
 
+func (a App) GetStakingKeeper() stakingkeeper.Keeper {
+	return *a.StakingKeeper
```

Comment:
avoid mixing pointers vs non-pointers. Some types are alwyas used as pointers, others are always used as non-pointers, try to stick with how the thing is used already.

Reply:
in general, use the types like you found them, try to avoid using `&` or `*` for existing values.